src/main.ts

```diff
@@ -20,6 +20,16 @@ import 'dayjs/locale/ru'
 const app = createApp(App)
 
 app.config.globalProperties.appVersion = packageJSON.version
+const isTouchable =  'ontouchstart' in window || navigator.maxTouchPoints > 0;
```

Comment:
Is is a correct module for this functions/vars?

Reply:
Merged https://github.com/Adamant-im/adamant-im/pull/563.
@PaulDremanovich merge the `dev` into current branch. You can now use the `isMobile()` from `lib/display-mobile.ts`.